integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java

```diff
@@ -665,278 +629,52 @@ void testIncludeServerOutInPodLog() {
-    checkEvent(opNamespace, domainNamespace3, null, NAMESPACE_WATCHING_STARTED, "Normal", timestamp);
-
-    // verify there is no event logged in domainNamespace4
-    logger.info("verify NamespaceWatchingStarted event is not logged in {0}", domainNamespace4);
-    assertFalse(domainEventExists(opNamespace, domainNamespace4, null, NAMESPACE_WATCHING_STARTED,
-        "Normal", timestamp), "domain event " + NAMESPACE_WATCHING_STARTED + " is logged in "
-        + domainNamespace4 + ", expected no such event will be logged");
-
-    timestamp = now();
-    logger.info("Labelling namespace {0} to \"weblogic-operator=disabled\" to disable it in the operator "
-        + "watch list", domainNamespace3);
-
-    // label domainNamespace3 to weblogic-operator=disabled
-    Command
-        .withParams(new CommandParams()
-            .command("kubectl label ns " + domainNamespace3 + " weblogic-operator=disabled --overwrite"))
-        .execute();
-
-    logger.info("verify NamespaceWatchingStopped event is logged in namespace {0}", domainNamespace3);
-    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace3, null, "Normal", timestamp,
-        enableClusterRoleBinding);
-
-    if (enableClusterRoleBinding) {
-      String newNSWithoutLabels = "ns-newnamespace1";
-      String newNSWithLabels = "ns-newnamespace2";
-
-      assertDoesNotThrow(() -> createNamespaces(newNSWithoutLabels, newNSWithLabels),
-          "Failed to create new namespaces");
-
-      Command
-          .withParams(new CommandParams()
-              .command("kubectl label ns " + newNSWithLabels + " weblogic-operator=enabled --overwrite"))
-          .execute();
-
-      logger.info("verify NamespaceWatchingStarted event is logged in namespace {0}", newNSWithLabels);
-      checkEvent(opNamespace, newNSWithLabels, null, NAMESPACE_WATCHING_STARTED, "Normal", timestamp);
-
-      // verify there is no event logged in domainNamespace4
-      logger.info("verify NamespaceWatchingStarted event is not logged in {0}", domainNamespace4);
-      assertFalse(domainEventExists(opNamespace, newNSWithoutLabels, null, NAMESPACE_WATCHING_STARTED,
-          "Normal", timestamp), "domain event " + NAMESPACE_WATCHING_STARTED + " is logged in "
-          + newNSWithoutLabels + ", expected no such event will be logged");
+    try {
+      OffsetDateTime timestamp = now();
+      createDomain(domainNamespace2, domainUid, pvName2, pvcName2);
+      deleteDomainCustomResource(domainUid, domainNamespace2);
+      checkPodDoesNotExist(adminServerPodName, domainUid, domainNamespace2);
+      checkPodDoesNotExist(managedServerPodNamePrefix + 1, domainUid, domainNamespace2);
+      checkPodDoesNotExist(managedServerPodNamePrefix + 2, domainUid, domainNamespace2);
+
+      //verify domain deleted event
+      checkEvent(opNamespace, domainNamespace2, domainUid, DOMAIN_DELETED, "Normal", timestamp);
+    } finally {
+      shutdownDomain(domainUid, domainNamespace2);
```

Comment:
What happens if you try to patch a domain resource that doesn't exist?

Reply:
fixed